handlers/handlers.go

```diff
@@ -5,18 +5,20 @@ import (
 	"io/ioutil"
 	"net/http"
 
+	"github.com/ONSdigital/dp-frontend-geography-controller/config"
 	"github.com/ONSdigital/dp-frontend-models/model/geographyHomepage"
 
 	"github.com/ONSdigital/dp-frontend-geography-controller/models"
 
-	"github.com/ONSdigital/go-ns/common"
 	"github.com/ONSdigital/go-ns/healthcheck"
 	"github.com/ONSdigital/go-ns/log"
 )
 
 const dataEndpoint = `\/data$`
 const localAuthority = `?type=geography`
 
+var url = config.Get().CodeListsAPIURL
```

Comment:
I would consider passing the config struct into the function/method that needs it instead of creating a variable.